tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/drop_cope_duplicate_responses_test.py

```diff
@@ -0,0 +1,85 @@
+"""
+Unit tests for drop_cope_duplicate_responses module
+
+Removes the duplicate sets of COPE responses to the same questions in the same survey version.
+"""
+import unittest
+
+from cdr_cleaner.cleaning_rules.drop_cope_duplicate_responses import (
+    DropCopeDuplicateResponses, COPE_SURVEY_VERSION_MAP_TABLE)
+from constants.cdr_cleaner import clean_cdr as clean_consts
+
+
+class DropCopeDuplicateResponsesTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+    def setUp(self):
+        self.project_id = 'foo_project'
+        self.dataset_id = 'foo_dataset'
+        self.sandbox_id = 'foo_sandbox'
+        self.client = None
+
+        self.rule_instance = DropCopeDuplicateResponses(self.project_id,
+                                                        self.dataset_id,
+                                                        self.sandbox_id)
+
+        self.assertEqual(self.rule_instance.project_id, self.project_id)
+        self.assertEqual(self.rule_instance.dataset_id, self.dataset_id)
+        self.assertEqual(self.rule_instance.sandbox_dataset_id, self.sandbox_id)
+
+    def test_get_query_specs(self):
+        # Pre conditions
+        self.assertEqual(self.rule_instance.affected_datasets,
+                         [clean_consts.RDR])
+
+        # Test
+        results_list = self.rule_instance.get_query_specs()
+        # Post conditions
+        sandbox_query = dict()
+        sandbox_query[
```

Comment:
Please update this test to generate queries in a different way other than calling the functions

Reply:
updated!